### ISSUE TITLE
Create individual pages for the 6 topics

### ISSUE THREAD

**Issue by Mahak008** [asdsyd/Healthy-developer#38]

**Comment by asdsyd** [comment]
![Screenshot 2022-10-17 at 11 24 33 PM](https://user-images.githubusercontent.com/77977076/196248291-9e5dc1b5-f00e-4e79-8d1d-e23a34abd5f9.png)

**Comment by asdsyd** [comment]
I'll spread this out with individual page in one issue so everyone gets a chance to contribute.

**Comment by asdsyd** [comment]
Before working on it. Lets discuss on how we plan to make it. 
As we want to keep the info short and crisp.
It can be something like pinterest cards with twitter handle of (senior) develop
Each card having an advise related to that specific topic.

**Comment by asdsyd** [comment]
[Scanned Documents.pdf](https://github.com/asdsyd/Healthy-developer/files/9803318/Scanned.Documents.pdf)

**Comment by asdsyd** [comment]
![Scanned Documents](https://user-images.githubusercontent.com/77977076/196250229-e12db45a-966d-471b-98df-51b77cce2b99.jpg)

**Comment by asdsyd** [comment]
Any ideas or suggestions drop down below 👇

**Comment by Mubashir-Md** [comment]
Hey Asad, you need individual page for each of them and with a little info about them on their page..right?

**Comment by asdsyd** [comment]
No just see the design above. It's card based twitter screenshots advise from senior devs

**Comment by asdsyd** [comment]
@Mubashir-Md You can also checkout my DSAwithKunal repo, it covers hacktoberfest contributions aswell.

**Comment by theshoonyam** [comment]
> No just see the design above. It's card based twitter screenshots advise from senior devs

Here senior devs means those people who are relevant in the industry, right?

**Comment by asdsyd** [comment]
Yeah. Experienced developers sharing their productivity advise or tip.

**Comment by theshoonyam** [comment]
I can work on it.

**Comment by asdsyd** [comment]
Please comment in respective issues so I can assign you. Thanks

**Comment by ozirichigozie** [comment]
# Hello @asdsyd 🙋‍♂️

If you can provide the photos, I can help you work on the pages.

I'll be using `JavaScript`, `HTML` and `CSS`, the cards for each topic will appear `onclick`.

I hope it's not too late to hacktoberfest this issue.

Thank you!

**Comment by asdsyd** [comment]
@ozirichigozie  You mean the content screenshots? Okay. Please comment in which page u want to work I'll assign you there.And you must use tailwind css for this. Thanks.
